### PR TITLE
feat: Add content to blips

### PR DESCRIPTION
**Pull request by andrablaj**
Update content in the blips, such as links & CHT documentation.

### CODE REVIEW

File: radar/2023-09-12/k3d.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ quadrant:   platforms-and-aoe-services
 tags:       [deployment-management]
 ---
 
-Lighter version of Kubernetes for orchestration.
\ No newline at end of file
+k3d is a lighter version of Kubernetes for orchestration.
```
Comment (garethbowen) [inline radar/2023-09-12/k3d.md:8]:
```suggestion
k3d is a lightweight version of Kubernetes for orchestration.
```

File: radar/2023-09-12/klipfolio.md [hunk 2]
```diff
@@ -5,9 +5,6 @@ quadrant:   methods-and-patterns
 tags:       [data-use]
 ---
 
-Klipfolio is a paid third-party SAAS option for building dashboards.
+[Klipfolio](https://www.klipfolio.com/) is a paid third-party SAAS option for building dashboards.
```
Comment (garethbowen) [inline radar/2023-09-12/klipfolio.md:8]: I think we should move klipfolio out of adopt and into "stop", and add a recommendation to use Superset instead.

File: radar/2023-09-12/medic-os.md [hunk 3]
```diff
@@ -5,4 +5,4 @@ quadrant:   platforms-and-aoe-services
 tags:       [deployment-management]
 ---
 
-A space-efficient Linux-based operating system, designed specifically to run Medic's mHealth platform. This is the build system and source code; Docker images will be available on Medic's website.
\ No newline at end of file
+Medic OS is a space-efficient Linux-based operating system, designed specifically to run Medic's mHealth platform. This is the build system and source code; Docker images will be available on Medic's website.
```
Comment (garethbowen) [inline radar/2023-09-12/medic-os.md:8]: For items in "stop" I think we should aim to include an explanation of why stop them... eg:

```suggestion
Medic OS is a space-efficient Linux-based operating system, designed specifically to run Medic's mHealth platform. This is the build system and source code; Docker images will be available on Medic's website.

From CHT Core Framework 4.0.0 onwards the Docker image is based on Alpine, a third party lightweight linux OS, so Medic OS is now deprecated.
```

File: radar/2023-09-12/klipfolio.md (hunk 2 above)
Comment (andrablaj) [reply radar/2023-09-12/klipfolio.md]: Good point! I will make the same change in the Contributors' tech radar too.

**Review by garethbowen — APPROVED** [review]
Nice one.

What is going on with the build though??

**Pull request merged**